iroh-store/src/rpc.rs

```diff
@@ -18,107 +16,131 @@ impl iroh_rpc_types::NamedService for Store {
     const NAME: &'static str = "store";
 }
 
-#[async_trait]
-impl RpcStore for Store {
+#[derive(Clone)]
+pub struct RpcStore(Store);
```

Comment:
why do we need a new type and don't just implement this directly on the Store?

Reply:
Also, as you have probably seen from the code so far, I don't mind newtypes that much... :-)